lib/manageiq_foreman/lib/manageiq_foreman/connection.rb

```diff
@@ -1,12 +1,12 @@
 module ManageiqForeman
   class Connection
     CLASSES = {
-      :config_templates  => ForemanApi::Resources::ConfigTemplate,
+      :configtemplates   => ForemanApi::Resources::ConfigTemplate,
```

Comment:
Why remove the underscores?  I like that the key matches the .underscore of the value....makes it consistent with other parts of the code.